index.html

```diff
@@ -47,6 +47,148 @@ <h2 class="secondary-text">I'am a Software Developer</h2>
+        </div>
+        <div class="content-defaults content2">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content3">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content4">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content5">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+        <div class="content-defaults content6">
+          <h2 class="other-content-title">Profesional Art Printing Data</h2>
+          <p class="other-content-description">A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard</p>
+          <ul class="other-content-tags">
+            <li>html</li>
+            <li>bootstrap</li>
+            <li>Ruby</li>
+          </ul>
+          <a class="other-content-link" href="#" target="_blank">See Project</a>
+        </div>
+      </div>
+    </div>
+    <div class="about">
+      <div class="about-top">
+        <h1 class="about-primary-text">About me</h1>
```

Comment:
I suggest you change this to any other `<hx> tag from <h2> to. <6>`
There can only be one `<h1>` tag in a page for browser SEO optimization and for screen readers.
read more [here](https://ahrefs.com/blog/h1-tag/#:~:text=skim%20and%20digest.-,H1%20tags%20improve%20accessibility,title%20be%20an%20H1%20tag.)